keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -726,26 +726,24 @@ kpxc.fillInFromActiveElement = async function(passOnly = false) {
         return;
     }
 
-    const el = document.activeElement;
-    if (el.nodeName !== 'INPUT') {
-        // No active input element selected -> fill the first combination found
-        if (kpxc.combinations.length > 0) {
+    if (kpxc.combinations.length > 0 && kpxc.settings.autoCompleteUsernames) {
+        const field = passOnly ? kpxc.combinations[0].password : kpxc.combinations[0].username;
+        // set focus to the input field
+        field.focus();
```

Comment:
Need to check for `null` here

Reply:
can it be null if `kpxc.combinations.length > 0`?

Reply:
The `null` check is not enough here. At the Google login page the password field is actually in the second `combination`. So if `passOnly` is used, maybe all input fields from the combinations must be checked until a password field is found. If not, then just return.

Ugly almost-like-this-code:
```
        let field;
        if (passOnly) {
            const passwordField = kpxc.combinations.find(c => c.password);
            if (passwordField && passwordField.password) {
                field = passwordField.password;
            }
        } else {
            field = kpxc.combinations[0].username;
        }

        if (!field) {
            return;
        }
```